src/Arcus.Testing.Tests.Unit/Assert_/AssertCsvTests.cs

```diff
@@ -240,6 +240,61 @@ public void CompareWithIgnoredRowAndColumnOrder_WithShuffledCsv_Succeeds()
             });
         }
 
+        [Property]
+        public void CompareWithIgnoredColumnOrderAndColumnIndexes_WithShuffledCsv_FailsWithDescription()
+        {
+            // Arrange
+            TestCsv expected = TestCsv.Generate();
+            TestCsv actual = expected.Copy();
+            int[] possibleIndexes = Enumerable.Range(0, expected.ColumnCount).ToArray();
+            int ignoredIndex = Bogus.PickRandom(possibleIndexes);
+
+            actual.ShuffleColumns();
+
+            CompareShouldFailWithDescription(
+                expected,
+                actual,
+                options =>
+                {
+                    options.ColumnOrder = AssertCsvOrder.Ignore;
+                    options.IgnoreColumnIndex(ignoredIndex);
```

Comment:
Already a good number of tests to handle this case, was just thinking of a test that would check that if we ignore the same column both by name and index, it still succeeds.

Reply:
Added test case here [6b0f144](https://github.com/arcus-azure/arcus.testing/pull/161/commits/6b0f144e63dc74b3d3cc5b1ad9dbdd9c32117eff)